tarpaulin-install/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'tarpaulin-installer'
+description: 'Install tarpaulin and includes it inside of PATH'
+branding:
+  icon: 'package'
+  color: 'blue'
+inputs:
+  VERSION:
+    description: 'tarpaulin release to be installed'
+    required: false
+    default: "0.27.3"
```

Comment:
In this case we have the current latest version hard-coded. I would drop this input variable and use the code snippet from below